server/server.go

```diff
@@ -80,35 +64,38 @@ func NewValkyrie(ctx context.Context, cfg *configs.ValkyrieConfig) *Valkyrie {
 		TraceConfig: cfg.Telemetry.Tracing,
 	})
 	if err != nil {
-		log.Fatal().Err(err).Msg("Error getting pam client")
+		log.Err(err).Msg("Error getting pam client")
+		return nil, err
 	}
 
 	// Provider routes.
 	if err = routes.ProviderRoutes(v.provider, cfg, pamClient, httpClient); err != nil {
-		log.Fatal().Err(err).Msg("Unable to setup the intended provider routes")
+		log.Err(err).Msg("Unable to setup the intended provider routes")
+		return nil, err
 	}
 	if err = routes.OperatorRoutes(v.operator, cfg, httpClient); err != nil {
-		log.Fatal().Err(err).Msg("Unable to setup the intended operator routes")
+		log.Err(err).Msg("Unable to setup the intended operator routes")
```

Comment:
Similar to previous comment, is it fine to omit the logging level?

Reply:
yes